src/constants.ts

```diff
@@ -0,0 +1,13 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+/*
+Starting off with a small set of constants used in multiple places. This should hopefully become something akin to a resource file for localization.
```

Comment:
I would remove this comment. Ideally we don't use the same strings over and over again, so we won't need 'something akin to a resource file'

Typescript localization looks like this: https://github.com/Microsoft/vscode-azurefunctions/blob/master/src/extension.ts#L30

It's actually better than a resource file because the english string is in-line with the code (and thus easier to read/understand)